packages/waku/src/lib/builder/build.ts

```diff
@@ -144,7 +144,7 @@ const analyzeEntries = async (rootDir: string, config: ConfigDev) => {
         build: {
           write: false,
           ssr: true,
-          target: 'node18',
+          target: 'node22',
```

Comment:
Ah, I was going to do this originally, but forgot about it.

Do they have `node22` for the target option? We still support Node 20, so it doesn't seem correct.

Reply:
https://vite.dev/config/build-options#build-target

https://esbuild.github.io/api/#target

`node20` seems good.